common.go

```diff
@@ -3,7 +3,6 @@ package anthropic
 type Model string
 
 const (
-	ModelClaudeInstant1Dot2        Model = "claude-instant-1.2"
```

Comment:
I think it would be better not to delete it outright, but to mark it as Deprecated

Reply:
Possibly - I just think if they're going to suddenly vanish on November 6 we should probably delete them then.
I think this would make sense to merge on November 6 as the PR stands, but if you want to have a version before then that prints a deprecation warning on use of Claude 1 models, then that's more involved. Given that that's two weeks away, I don't think that is worth the amount of work?

Reply:
I agree, I will merge this PR on November 6th